onnxruntime/test/util/default_providers.cc

```diff
@@ -136,7 +136,7 @@ std::unique_ptr<IExecutionProvider> DefaultNnapiExecutionProvider() {
 // For any non - Android system, NNAPI will only be used for ort model converter
 // Make it unavailable here, you can still manually append NNAPI EP to session for model conversion
 #if defined(USE_NNAPI) && defined(__ANDROID__)
-  return CreateExecutionProviderFactory_Nnapi(0, {})->CreateProvider();
+  return CreateExecutionProviderFactory_Nnapi(2, {})->CreateProvider();
```

Comment:
sort of temporary solution as a hack to add use_nchw when creating default execution provider for optest to let DepthToSpace UT passes.

will look into try to add `DepthToSpace` to layout sensitive ops to handle in layout optimizer later.

Reply:
could we use `NNAPI_FLAG_USE_NCHW`? and maybe add a comment about why, even if it is temporary?